lib/better_params.ex

```diff
@@ -13,11 +13,12 @@ defmodule BetterParams do
   @doc """
   Initializes the Plug.
 
-  This plug doesn't take any arguments.
+
   """
-  @spec init(term :: term) :: :ok
-  def init(_) do
-    :ok
+  @spec init(term :: term) :: {boolean}
+  def init(opts) do
+    drop_string_keys = Keyword.get(opts, :drop_string_keys, false)
+    {drop_string_keys}
```

Comment:
This is more of a personal preference, but I believe instead of parsing the options here, the whole keyword list should be passed as-is to the `symbolize_merge/2` method, and the `drop_string_keys` value should be checked in there (since it's the only internal method doing the heavy lifting).

Also, the typespec for keyword lists is `Keyword.t`:

```elixir
@spec init(opts :: Keyword.t) :: Keyword.t
```